# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/tests/shaders/initialization.vert
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/tests/shaders/initialization.vert b/third_party/rust/spirv-cross-internal/tests/shaders/initialization.vert
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/tests/shaders/initialization.vert
+++ /dev/null
@@ -1,14 +0,0 @@
-#version 310 es
-
-layout(location = 0) in float rand;
-
-void main()
-{
-    vec4 pos;
-
-    if (rand > 0.5) {
-        pos = vec4(1.0);
-    }
-
-    gl_Position = pos;
-}